client/src/main/java/org/apache/celeborn/client/ShuffleClientImpl.java

```diff
@@ -47,8 +46,7 @@
 import org.apache.celeborn.common.network.client.RpcResponseCallback;
 import org.apache.celeborn.common.network.client.TransportClient;
 import org.apache.celeborn.common.network.client.TransportClientFactory;
-import org.apache.celeborn.common.network.protocol.PushData;
-import org.apache.celeborn.common.network.protocol.PushMergedData;
+import org.apache.celeborn.common.network.protocol.*;
```

Comment:
ditto

Reply:
it's optimized automaticly by code format